README.md

```diff
@@ -1,24 +1,36 @@
-# Partgui
+# Particl-UI
+
+![UI Preview](preview.png)
 
 [![Build Status](https://travis-ci.org/particl/partgui.svg?branch=master)](https://travis-ci.org/particl/partgui)
 [![Coverage Status](https://coveralls.io/repos/github/particl/partgui/badge.svg?branch=master)](https://coveralls.io/github/particl/partgui?branch=master)
 [![Code Climate](https://codeclimate.com/github/particl/partgui/badges/gpa.svg)](https://codeclimate.com/github/particl/partgui)
 [![Greenkeeper badge](https://badges.greenkeeper.io/particl/partgui.svg)](https://greenkeeper.io/)
 
+> *"Particl is an open source project that aims to restore the balance of privacy on the internet."* 
+
+We provide a decentralized privacy platform with a suite of tools to enhance your online privacy:
+* **An anonymous cryptocurrency** - send and receive the PART cryptocurrency without revealing the transaction history
+* **End-to-end encrypted messaging** - communicate in a secure and decentralized manner without revealing your IP address
```

Comment:
This will only come after the market place... We aren't putting any work into this until after primary